pages/[[server]]/index.vue

```diff
@@ -0,0 +1,15 @@
+<script setup lang="ts">
+const instance = instanceStorage.value[currentServer.value]
+try {
+  clearError({ redirect: currentUser.value ? '/home' : `/${currentServer.value}/public/local` })
```

Comment:
This will redirect to the public timeline when visitor opens /[server] route. Currently it is just an error page 😄